test/plugin_functional/test_suites/data_plugin/session.ts

```diff
@@ -115,6 +119,19 @@ export default function ({ getService, getPageObjects }: PluginFunctionalProvide
       });
 
       it('starts a session on filter change', async () => {
+        // For some reason, when loading the dashboard, sometimes the filter doesn't show up, so we
+        // refresh until it shows up
```

Comment:
Hmm it sounds like this may be catching an actual issue in Dashboard. I wonder if anyone from @elastic/kibana-presentation has an idea of why this started failing a couple of weeks ago?

Reply:
Created https://github.com/elastic/kibana/issues/194307 to track

Reply:
Great, thanks. Created a followup issue on our end to revert this change once the Dashboard issue is resolved: #194331.